src/lib/stepper/_stepper-theme.scss

```diff
@@ -33,11 +33,11 @@
     background-color: mat-color($background, card);
   }
 
-  .vertical-content-container {
+  .mat-vertical-content-container {
     border-left-color: mat-color($foreground, divider);
   }
 
-  .connector-line {
+  .mat-connector-line {
```

Comment:
before this gets merged into master you should go through and make sure the classes all start with `mat-stepper-` or `mat-step-`